Livecharts crash WindowsSdkPackageVersion 10.0.22621.38+

**Describe the bug**
 When loading the Livecharts in XAML in an App with WindowsSdkPackageVersion 10.0.22621.38 or higher it crashes the App and Debugger. WindowsSdkPackageVersion 10.0.22621.34 and lower work as expected

I encountered this when I wanted to update CommunityToolkit.Mvvm from 8.2.2 to 8.3.0, I had to increase WindowsSdkPackageVersion (Microsoft.Windows.CsWinRT 2.1.1 also requires a higher version!)

CommunityToolkit.Mvvm 8.3.0 features many changes like .net8 and other things were made optional.
https://devblogs.microsoft.com/dotnet/announcing-the-dotnet-community-toolkit-830/

**To Reproduce**
Steps to reproduce the behavior:
1. Set <WindowsSdkPackageVersion>10.0.22621.38</WindowsSdkPackageVersion> in Solution
2. Make a XAML page with a livechart
```xml
<Page
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:lvc="using:LiveChartsCore.SkiaSharpView.WinUI"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">

    <Grid>
        <lvc:CartesianChart />
    </Grid>
</Page>
```
3. Run the app / open the Page
4. See App and debugger crash, it didn't even save an unhandled exception

**Desktop (please complete the following information):**
 - OS: Windows 11 Build 22631
 - Build 22631

**Additional context**
LiveChartsCore.SkiaSharpView.WinUI 2.0.0-rc2.1
net8.0-windows10.0.22621.0
Microsoft.WindowsAppSDK 1.5.240802000

I there, 

Face the same issue when updating the CommunityToolkit.Mvvm to version 8.3.2 and try to generate the packages from the skiasharp-3 branch but unfortunately without any success.

After cloning the repo locally, switch to the skiasharp-3 and adapted the projects configurations to only build the WinUi version of the package. 

When using the provided scripts, I'm able to generate the packages but when I use them in our application, I received the following exception:
> Exception thrown: 'Microsoft.UI.Xaml.Markup.XamlParseException' in WinRT.Runtime.dll
WinRT information: Cannot locate resource from 'ms-appx:///LiveChartsCore.SkiaSharpView.WinUI/CartesianChart.xaml'.
XAML parsing failed.

I then tried to generate the package from Visual Studio 17.11.5 but get the following build error in Release:
> src\skiasharp\LiveChartsCore.SkiaSharp\VisualElements\SVGVisual.cs(45,38,45,51): error CS0507: 'SVGVisual.OnInvalidated(Chart<SkiaSharpDrawingContext>)': cannot change access modifiers when overriding 'protected internal' inherited member 'GeometryVisual<SVGPathGeometry, LabelGeometry, SkiaSharpDrawingContext>.OnInvalidated(Chart<SkiaSharpDrawingContext>)'

Switching back the configuration to Debug solved this issue but lead to the first exception when generating the packages.

Do you have any advice on how to generate a package that contains the fix?

--- a/samples/WinUISample/WinUISample/WinUISample.csproj
+++ b/samples/WinUISample/WinUISample/WinUISample.csproj
@@ -5,15 +5,12 @@
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <RootNamespace>WinUISample</RootNamespace>
     <ApplicationManifest>app.manifest</ApplicationManifest>
-    <Platforms>x86;x64;arm64</Platforms>
-    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <Platforms>x86;x64;ARM64</Platforms>
+    <RuntimeIdentifiers Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) &gt;= 8">win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <RuntimeIdentifiers Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) &lt; 8">win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
     <PublishProfile>win10-$(Platform).pubxml</PublishProfile>
     <UseWinUI>true</UseWinUI>
     <EnableMsixTooling>true</EnableMsixTooling>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <WindowsSdkPackageVersion>10.0.19041.38</WindowsSdkPackageVersion>
   </PropertyGroup>
 
   <ItemGroup>
@@ -38,6 +35,16 @@
   <ItemGroup Condition="'$(DisableMsixProjectCapabilityAddedByProject)'!='true' and '$(EnablePreviewMsixTooling)'=='true'">
     <ProjectCapability Include="Msix" />
   </ItemGroup>
+
+  <!--
+    Defining the "HasPackageAndPublishMenuAddedByProject" property here allows the Solution
+    Explorer "Package and Publish" context menu entry to be enabled for this project even if
+    the Windows App SDK Nuget package has not yet been restored.
+  -->
+  <PropertyGroup Condition="'$(DisableHasPackageAndPublishMenuAddedByProject)'!='true' and '$(EnableMsixTooling)'=='true'">
+    <HasPackageAndPublishMenu>true</HasPackageAndPublishMenu>
+  </PropertyGroup>
+  
   <ItemGroup>
     <ProjectReference Include="..\..\..\src\skiasharp\LiveChartsCore.SkiaSharpView.WinUI\LiveChartsCore.SkiaSharpView.WinUI.csproj" />
     <ProjectReference Include="..\..\ViewModelsSamples\ViewModelsSamples.csproj" />

--- a/samples/WinUISample/WinUISample/WinUISample.csproj
+++ b/samples/WinUISample/WinUISample/WinUISample.csproj
@@ -9,7 +9,7 @@
     <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
     <PublishProfile>win10-$(Platform).pubxml</PublishProfile>
     <UseWinUI>true</UseWinUI>
-    <EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
+    <EnableMsixTooling>true</EnableMsixTooling>
   </PropertyGroup>
 
   <ItemGroup>
@@ -23,8 +23,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.3.230602002" />
-    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240923002" />
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
     <Manifest Include="$(ApplicationManifest)" />
   </ItemGroup>
 

--- a/samples/WinUISample/WinUISample/WinUISample.csproj
+++ b/samples/WinUISample/WinUISample/WinUISample.csproj
@@ -12,6 +12,10 @@
     <EnableMsixTooling>true</EnableMsixTooling>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <WindowsSdkPackageVersion>10.0.19041.38</WindowsSdkPackageVersion>
+  </PropertyGroup>
+
   <ItemGroup>
     <Content Include="Assets\SplashScreen.scale-200.png" />
     <Content Include="Assets\LockScreenLogo.scale-200.png" />

--- a/src/LiveChartsCore.Behaviours/LiveChartsCore.Behaviours.csproj
+++ b/src/LiveChartsCore.Behaviours/LiveChartsCore.Behaviours.csproj
@@ -10,7 +10,6 @@
     </TargetFrameworks>
     <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">
       $(TargetFrameworks);
-      net6.0-windows10.0.19041.0;
       net8.0-windows10.0.19041.0;
     </TargetFrameworks>
 

--- a/src/LiveChartsCore/LiveChartsCore.csproj
+++ b/src/LiveChartsCore/LiveChartsCore.csproj
@@ -11,7 +11,6 @@
     <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">
       $(TargetFrameworks);
       net462;
-      net6.0-windows10.0.19041.0;
       net8.0-windows10.0.19041.0;net8.0-windows10.0.20348.0;
     </TargetFrameworks>
 

--- a/src/skiasharp/LiveChartsCore.SkiaSharp.WPF/LiveChartsCore.SkiaSharpView.WPF.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp.WPF/LiveChartsCore.SkiaSharpView.WPF.csproj
@@ -30,7 +30,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="SkiaSharp.Views.WPF" Version="2.88.8" />
+    <PackageReference Include="SkiaSharp.Views.WPF" Version="3.0.0-preview.4.1" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharp.WinForms/LiveChartsCore.SkiaSharpView.WinForms.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp.WinForms/LiveChartsCore.SkiaSharpView.WinForms.csproj
@@ -38,7 +38,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="SkiaSharp.Views.WindowsForms" Version="2.88.8" />
+    <PackageReference Include="SkiaSharp.Views.WindowsForms" Version="3.0.0-preview.4.1" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharp.Xamarin.Forms/LiveChartsCore.SkiaSharpView.Xamarin.Forms.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp.Xamarin.Forms/LiveChartsCore.SkiaSharpView.Xamarin.Forms.csproj
@@ -29,7 +29,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
-    <PackageReference Include="SkiaSharp.Views.Forms" Version="2.88.8" />
+    <PackageReference Include="SkiaSharp.Views.Forms" Version="3.0.0-preview.4.1" />
     <PackageReference Include="Xamarin.Essentials" Version="1.6.0" />
     <PackageReference Include="Xamarin.Forms" Version="5.0.0.1874" />
   </ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
@@ -49,9 +49,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="SkiaSharp" Version="3.0.0-preview.4.1" />
-    <PackageReference Include="SkiaSharp.HarfBuzz" Version="3.0.0-preview.4.1" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+    <PackageReference Include="SkiaSharp" Version="3.0.0-preview.5.3" />
+    <PackageReference Include="SkiaSharp.HarfBuzz" Version="3.0.0-preview.5.3" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
@@ -36,7 +36,7 @@
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
-    <DebugType>embedded</DebugType>
+    <DebugType>embedded</DebugType
   </PropertyGroup>
 
   <PropertyGroup>
@@ -50,8 +50,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
-    <PackageReference Include="SkiaSharp" Version="3.0.0-preview.5.3" />
-    <PackageReference Include="SkiaSharp.HarfBuzz" Version="3.0.0-preview.5.3" />
+    <PackageReference Include="SkiaSharp" Version="2.88.8" />
+    <PackageReference Include="SkiaSharp.HarfBuzz" Version="2.88.8" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
@@ -11,7 +11,6 @@
     <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">
       $(TargetFrameworks);
       net462;
-      net6.0-windows10.0.19041.0;
       net8.0-windows10.0.19041.0;net8.0-windows10.0.20348.0;
     </TargetFrameworks>
 

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
@@ -36,7 +36,7 @@
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
-    <DebugType>embedded</DebugType
+    <DebugType>embedded</DebugType>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/LiveChartsCore.SkiaSharpView.csproj
@@ -51,8 +51,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="SkiaSharp" Version="2.88.8" />
-    <PackageReference Include="SkiaSharp.HarfBuzz" Version="2.88.8" />
+    <PackageReference Include="SkiaSharp" Version="3.0.0-preview.4.1" />
+    <PackageReference Include="SkiaSharp.HarfBuzz" Version="3.0.0-preview.4.1" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/Painting/ImageFilters/Blur.cs
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/Painting/ImageFilters/Blur.cs
@@ -35,10 +35,12 @@ namespace LiveChartsCore.SkiaSharpView.Painting.ImageFilters;
 /// <param name="sigmaX">The sigma x.</param>
 /// <param name="sigmaY">The sigma y.</param>
 /// <param name="input">The input.</param>
-/// <param name="cropRect">The crop rect.</param>
-public class Blur(float sigmaX, float sigmaY, SKImageFilter? input = null, SKImageFilter.CropRect? cropRect = null) : ImageFilter
+public class Blur(
+    float sigmaX,
+    float sigmaY,
+    SKImageFilter? input = null)
+        : ImageFilter
 {
-
     /// <summary>
     /// Clones this instance.
     /// </summary>
@@ -46,7 +48,7 @@ public class Blur(float sigmaX, float sigmaY, SKImageFilter? input = null, SKIma
     /// <exception cref="System.NotImplementedException"></exception>
     public override ImageFilter Clone()
     {
-        return new Blur(sigmaX, sigmaY, input, cropRect);
+        return new Blur(sigmaX, sigmaY, input);
     }
 
     /// <summary>
@@ -56,6 +58,6 @@ public class Blur(float sigmaX, float sigmaY, SKImageFilter? input = null, SKIma
     /// <returns></returns>
     public override void CreateFilter(SkiaSharpDrawingContext drawingContext)
     {
-        SKImageFilter = SKImageFilter.CreateBlur(sigmaX, sigmaY, input, cropRect);
+        SKImageFilter = SKImageFilter.CreateBlur(sigmaX, sigmaY, input);
     }
 }

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/Painting/ImageFilters/DropShadow.cs
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/Painting/ImageFilters/DropShadow.cs
@@ -38,8 +38,14 @@ namespace LiveChartsCore.SkiaSharpView.Painting.ImageFilters;
 /// <param name="sigmaY">The sigma y.</param>
 /// <param name="color">The color.</param>
 /// <param name="input">The input.</param>
-/// <param name="cropRect">The crop rect.</param>
-public class DropShadow(float dx, float dy, float sigmaX, float sigmaY, SKColor color, SKImageFilter? input = null, SKImageFilter.CropRect? cropRect = null) : ImageFilter
+public class DropShadow(
+    float dx,
+    float dy,
+    float sigmaX,
+    float sigmaY,
+    SKColor color,
+    SKImageFilter? input = null)
+        : ImageFilter
 {
 
     /// <summary>
@@ -49,7 +55,7 @@ public class DropShadow(float dx, float dy, float sigmaX, float sigmaY, SKColor 
     /// <exception cref="System.NotImplementedException"></exception>
     public override ImageFilter Clone()
     {
-        return new DropShadow(dx, dy, sigmaX, sigmaY, color, input, cropRect);
+        return new DropShadow(dx, dy, sigmaX, sigmaY, color, input);
     }
 
     /// <summary>
@@ -58,6 +64,6 @@ public class DropShadow(float dx, float dy, float sigmaX, float sigmaY, SKColor 
     /// <param name="drawingContext">The drawing context.</param>
     public override void CreateFilter(SkiaSharpDrawingContext drawingContext)
     {
-        SKImageFilter = SKImageFilter.CreateDropShadow(dx, dy, sigmaX, sigmaY, color, input, cropRect);
+        SKImageFilter = SKImageFilter.CreateDropShadow(dx, dy, sigmaX, sigmaY, color, input);
     }
 }

--- a/src/skiasharp/LiveChartsCore.SkiaSharp/Painting/ImageFilters/ImageFiltersMergeOperation.cs
+++ b/src/skiasharp/LiveChartsCore.SkiaSharp/Painting/ImageFilters/ImageFiltersMergeOperation.cs
@@ -33,8 +33,7 @@ namespace LiveChartsCore.SkiaSharpView.Painting.ImageFilters;
 /// Initializes a new instance of the <see cref="ImageFiltersMergeOperation"/> class.
 /// </remarks>
 /// <param name="imageFilters">The image filters.</param>
-/// <param name="cropRect">The crop rect.</param>
-public class ImageFiltersMergeOperation(ImageFilter[] imageFilters, SKImageFilter.CropRect? cropRect = null) : ImageFilter
+public class ImageFiltersMergeOperation(ImageFilter[] imageFilters) : ImageFilter
 {
     private readonly ImageFilter[] _filters = imageFilters;
 
@@ -45,7 +44,7 @@ public class ImageFiltersMergeOperation(ImageFilter[] imageFilters, SKImageFilte
     /// <exception cref="System.NotImplementedException"></exception>
     public override ImageFilter Clone()
     {
-        return new ImageFiltersMergeOperation(_filters, cropRect);
+        return new ImageFiltersMergeOperation(_filters);
     }
 
     /// <summary>
@@ -65,7 +64,7 @@ public class ImageFiltersMergeOperation(ImageFilter[] imageFilters, SKImageFilte
             imageFilters[i++] = item.SKImageFilter;
         }
 
-        SKImageFilter = SKImageFilter.CreateMerge(imageFilters, cropRect);
+        SKImageFilter = SKImageFilter.CreateMerge(imageFilters);
     }
 
     /// <summary>

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.Blazor/LiveChartsCore.SkiaSharpView.Blazor.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.Blazor/LiveChartsCore.SkiaSharpView.Blazor.csproj
@@ -56,7 +56,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="SkiaSharp.Views.Blazor" Version="2.88.8" />
+    <PackageReference Include="SkiaSharp.Views.Blazor" Version="3.0.0-preview.4.1" />
     <PackageReference Include="HarfBuzzSharp.NativeAssets.WebAssembly" Version="7.3.0" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
 

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.Maui/LiveChartsCore.SkiaSharpView.Maui.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.Maui/LiveChartsCore.SkiaSharpView.Maui.csproj
@@ -43,8 +43,8 @@
 	</PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.3" />
-	  <PackageReference Include="SkiaSharp.Views.Maui.Controls.Compatibility" Version="3.0.0-preview.4.1" />
+    <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.82" />
+	  <PackageReference Include="SkiaSharp.Views.Maui.Controls" Version="3.0.0-preview.5.4" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
 	</ItemGroup>
 

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.Maui/LiveChartsCore.SkiaSharpView.Maui.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.Maui/LiveChartsCore.SkiaSharpView.Maui.csproj
@@ -44,7 +44,7 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.3" />
-	  <PackageReference Include="SkiaSharp.Views.Maui.Controls.Compatibility" Version="2.88.8" />
+	  <PackageReference Include="SkiaSharp.Views.Maui.Controls.Compatibility" Version="3.0.0-preview.4.1" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
 	</ItemGroup>
 

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.Uno.WinUI/LiveChartsCore.SkiaSharpView.Uno.WinUI.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.Uno.WinUI/LiveChartsCore.SkiaSharpView.Uno.WinUI.csproj
@@ -54,16 +54,16 @@
     <ProjectReference Include="..\..\LiveChartsCore.Behaviours\LiveChartsCore.Behaviours.csproj" />
     
     <!-- if uno then add reference to uno dependencies -->
-    <PackageReference Condition="!$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.8" />
+    <PackageReference Condition="!$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.Uno.WinUI" Version="3.0.0-preview.4.1" />
 
     <!-- in windows then use the winui package -->
-    <PackageReference Condition="$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.WinUI" Version="2.88.8" />
+    <PackageReference Condition="$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.4.1" />
   </ItemGroup>
   <Choose>
     <When Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
       <ItemGroup>
-        <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.5" />
-        <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22000.196" />
+        <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.221209.1" />
+        <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.1" />
       </ItemGroup>
     </When>
     <Otherwise>

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.Uno.WinUI/LiveChartsCore.SkiaSharpView.Uno.WinUI.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.Uno.WinUI/LiveChartsCore.SkiaSharpView.Uno.WinUI.csproj
@@ -48,21 +48,21 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
 
-    <PackageReference Condition="!$(TargetFramework.Contains('-windows'))" Include="Uno.WinUI" Version="5.0.19" />
+    <PackageReference Condition="!$(TargetFramework.Contains('-windows'))" Include="Uno.WinUI" Version="5.2.175" />
 
     <ProjectReference Include="..\LiveChartsCore.SkiaSharp\LiveChartsCore.SkiaSharpView.csproj" />
     <ProjectReference Include="..\..\LiveChartsCore.Behaviours\LiveChartsCore.Behaviours.csproj" />
     
     <!-- if uno then add reference to uno dependencies -->
-    <PackageReference Condition="!$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.Uno.WinUI" Version="3.0.0-preview.4.1" />
+    <PackageReference Condition="!$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.Uno.WinUI" Version="3.0.0-preview.5.4" />
 
     <!-- in windows then use the winui package -->
-    <PackageReference Condition="$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.4.1" />
+    <PackageReference Condition="$(TargetFramework.Contains('-windows'))" Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.5.4" />
   </ItemGroup>
   <Choose>
     <When Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
       <ItemGroup>
-        <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.221209.1" />
+        <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230913002" />
         <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.1" />
       </ItemGroup>
     </When>

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
@@ -47,9 +47,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.221209.1" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230913002" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.1" />
-    <PackageReference Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.4.1" />
+    <PackageReference Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.5.3" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
@@ -4,7 +4,6 @@
     <Nullable>enable</Nullable>
 
     <TargetFrameworks>
-      net6.0-windows10.0.19041.0;
       net8.0-windows10.0.19041.0;net8.0-windows10.0.20348.0;
     </TargetFrameworks>
 
@@ -50,7 +49,7 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.5" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22000.196" />
-    <PackageReference Include="SkiaSharp.Views.WinUI" Version="2.88.8" />
+    <PackageReference Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.4.1" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
@@ -10,9 +10,9 @@
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
 
     <EnableMsixTooling>true</EnableMsixTooling>
-    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>    
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
     <UseWinUI>true</UseWinUI>
-    
+
     <RootNamespace>LiveChartsCore.SkiaSharpView.WinUI</RootNamespace>
     <Version>2.0.0-rc3.3</Version>
     <PackageIcon>icon.png</PackageIcon>
@@ -47,8 +47,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.5" />
-    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22000.196" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.221209.1" />
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.1" />
     <PackageReference Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.4.1" />
   </ItemGroup>
 

--- a/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
+++ b/src/skiasharp/LiveChartsCore.SkiaSharpView.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj
@@ -49,7 +49,7 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230913002" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.1" />
-    <PackageReference Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.5.3" />
+    <PackageReference Include="SkiaSharp.Views.WinUI" Version="3.0.0-preview.5.4" />
   </ItemGroup>
 
   <ItemGroup>

